monitors_test.go

```diff
@@ -218,6 +236,18 @@ const monitorsjson = `
       "maxCheckAttempts": 5,
       "notificationInterval": 60
     },
+    {
+      "id"  : "2cSZzK3XfmG",
+      "type": "service",
+      "name": "Hatena-Blog - access_num.5xx_count",
+      "service": "Hatena-Blog",
+      "duration": 3,
+      "metric": "access_num.5xx_count",
+      "operator": ">",
+      "critical": 100.0,
+      "maxCheckAttempts": 3,
+      "notificationInterval": 60
+    },
```

Comment:
It's difficult to understand what is tested by these testcases.  I think it's better to comment each testcase mean,  but this is json string...